docs/content/guides/dagster/branch_depl_with_snowflake.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: Branch Deployments with Snowflake | Dagster
+description: This guide illustrates a workflow that enables testing Dagster code in your cloud environment without impacting your production data.
+---
+
+# Branch Deployments with Snowflake
+
+This guide details a workflow to test Dagster code in your cloud environment without impacting your production data. To highlight this functionality, we’ll leverage Dagster Cloud branch deployments and a Snowflake database to:
+
+- Execute code on a feature branch directly on Dagster Cloud
+- Read and write to a clone of our Snowflake data
+
+With these tools, we can merge changes with confidence in the impact on our data platform and with assurance that our code will execute as intended.
+
+### Prerequisites
+
```

Comment:
maybe a note about how branch deployments is a cloud only feature?

Reply:
Yeah I think we should make this very prominent so that people don't get an unpleasant surprise.  Maybe even include the Cloud logo somehow? cc @erinkcochran87 @braunjj for ideas on how to communicate this.

Reply:
I wouldn't include the logo - I think it'd feel distracting. IMO listing it in the prereqs like this ^ is enough.